crates/ef-testing/src/evm_sequencer/account/mod.rs

```diff
@@ -51,12 +51,34 @@ impl KakarotAccount {
     }
 }
 
-#[derive(Debug, Default, Clone)]
-pub enum AccountType {
-    #[default]
-    Uninitialized = 0,
-    EOA = 1,
-    Contract = 2,
+/// Analyze bytecode to collect all valid jumpdest indexes.
+/// Inspired from https://github.com/bluealloy/revm
+fn analyze(code: &[u8]) -> Vec<usize> {
```

Comment:
I'm sorry but I still find it absurd to copy paste a whole function just to slightly modify the return type. We can still use the `to_analyse` and just modify the return for our use. Or we can PR on Revm to make `analyze` pub.

Reply:
I tend to agree, we don't mind the extra types, that way we externalize as much code as possible to good crates such as revm

Reply:
ok, yeah I guess it doesn't make sense in rust because O(20000) is nothing...